src/Features/LanguageServer/Protocol/Extensions/Extensions.cs

```diff
@@ -89,16 +102,35 @@ public static ImmutableArray<DocumentId> GetDocumentIds(this Solution solution,
                 fileDocumentIds = solution.GetDocumentIdsWithFilePath(documentUri.LocalPath);
                 return fileDocumentIds;
             }
+            else if (documentUri.Scheme == SourceGeneratedFileScheme)
```

Comment:
no concern about case sensitivity? (my pref is def to keep like this, but i have a small amount of nervousness that some middleware is going to like UPPERCASE this).

Reply:
Not really, but we don't expect the client or middleware messing with casing in general, since we may be on a case-sensitive platform.